Benqi-Agents/DelegationMonitor/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "format": "^0.2.2",
+    "forta-agent": "^0.0.39",
+    "lru-cache": "^7.4.2"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/lru-cache": "^7.4.0",
+    "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.0.3",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
+    "prettier": "^2.6.0",
+    "ts-jest": "^27.0.3",
+    "typescript": "^4.3.4"
+  }
+}
```

Comment:
- Update `name` and `description` in package.json, seems like u erased the older version 

- I don't think the `format` dependency is needed 

-  `@types/lru-cache` should be under dependencies